### PR TITLE
binloginfo: use current time for unix socket filename

### PR DESCRIPTION
**Pull request by coocood**
To avoid potential conflict.

### CODE REVIEW

**Comment by tiancaiamao** [comment]
LGTM

File: sessionctx/binloginfo/binloginfo_test.go [hunk 1]
```diff
@@ -79,8 +80,7 @@ func (s *testBinlogSuite) SetUpSuite(c *C) {
 	c.Assert(err, IsNil)
 	s.store = store
 	tidb.SetSchemaLease(0)
-	s.unixFile = "/tmp/mock-binlog-pump"
-	os.Remove(s.unixFile)
```
Comment (zimulala) [inline sessionctx/binloginfo/binloginfo_test.go]: Why remove line 83?

Comment (coocood) [inline sessionctx/binloginfo/binloginfo_test.go]: As there will not be conflict, we do need to remove it first.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**